src/btree/bt_prefetch.c

```diff
@@ -59,8 +52,10 @@ __wt_btree_prefetch(WT_SESSION_IMPL *session, WT_REF *ref)
             ret = __wt_conn_prefetch_queue_push(session, next_ref);
             if (ret == 0)
                 ++block_preload;
-            else if (ret != EBUSY)
-                WT_RET(ret);
+            else if (ret != EBUSY) {
+                WT_STAT_CONN_INCR(session, block_prefetch_page_not_queued);
```

Comment:
I understand the intention of incrementing the `block_prefetch_page_not_queued ` here but do we also want to increment this stat if `ret == EBUSY`?

Reply:
Since I believed that running into a page that could not be pushed into the queue due to some checks was part of the 'expected' behaviour of prefetch, I didn't do the same in the case of `ret == EBUSY` since I thought it was not an error in the same way something returning a `WT_ERROR` here would be. I could extend it to increment the stat in both cases, though.

Reply:
Yeah that makes sense. Happy to leave this as is.